includes/polling/mempools/aruba-instant.inc.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * aruba-instant.inc.php
+ *
+ * LibreNMS mempools polling module for Aruba Instant
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 Timothy Willey
+ * @author     Timothy Willey <developer@timothywilley.net>
+ */
+echo 'aruba-instant-MEMORY-POOL: ';
+
+$memory_pool_total = snmpwalk_group($device, 'aiAPTotalMemory', 'AI-AP-MIB');
```

Comment:
It would be a little cleaner if you merged these walks together (with the fifth argument of snmpwalk_group)

Reply:
done